src/index.ts

```diff
@@ -109,7 +109,7 @@ export class SimpleNAT extends Resource {
   constructor(scope: Construct, id: string, props: SimpleNATProps) {
     super(scope, id);
 
-    let subnets;
+    var subnets;
```

Comment:
**Unexpected var, use let or const instead.**  [<sub>Why is this an issue?</sub>](http://sonarqube:9000/coding_rules?open=typescript:S3504&rule_key=typescript:S3504) 

![CODE_SMELL](https://hsonar.s3.ap-southeast-1.amazonaws.com/images/code_smell.png) Codesmell　　![CRITICAL](https://hsonar.s3.ap-southeast-1.amazonaws.com/images/critical.png) **Critical**　　:hourglass: *5min* effort